README.md

```diff
@@ -2,52 +2,71 @@
 
 Signal K Node server plugin to provide chart metadata, such as name, description and location of the actual chart tile data.
 
-Supports both v1 and v2 Signal K resources api paths.
+Chart metadata is derived from the following supported chart file types:
+- MBTiles _(.mbtiles)_
+- TMS _(tilemapresource.xml and tiles)_
+
+Additionally, chart metadata can be entered via the plugin configuration for other chart sources and types _(e.g. WMS, WMTS, S-57 tiles and tilejson)_.
+
+Chart metadata is made available to both v1 and v2 Signal K `resources` api paths.
 
 | Server Version | API | Path |
 |--- |--- |--- |
 | 1.x.x | v1 | `/signalk/v1/api/resources/charts` |
 | 2.x.x | v2 | `/signalk/v2/api/resources/charts` |
 
     
-_Note: v2 resource paths will only be made available on Signal K server >= v2._
+_Note: Version 2 resource paths will only be made available on Signal K server v2.0.0 and later_
 
-### Usage
+## Usage
 
-1. Install "Signal K Charts" plugin from Signal K Appstore
+1. Install `@signalk/signalk-charts` from the Signal K Server Appstore
 
-2. Configure plugin in **Plugin Config** 
+2. Configure the plugin in the Admin UI _(**Server -> Plugin Config -> Signal K Charts**)_ 
 
-- Add "Chart paths" which are the paths to the folders where chart files are stored. Defaults to `${signalk-configuration-path}/charts`
+3. Activate the plugin
 
+Chart metadata will then be available to client apps via the resources api `/resources/charts` for example:
+- [Freeboard SK](https://www.npmjs.com/package/@signalk/freeboard-sk)
+- [Tuktuk Chart Plotter](https://www.npmjs.com/package/tuktuk-chart-plotter)
 
-3. Add "Chart paths" in plugin configuration. Defaults to `${signalk-configuration-path}/charts`
 
-<img src="https://user-images.githubusercontent.com/1435910/39382493-57c1e4dc-4a6e-11e8-93e1-cedb4c7662f4.png" alt="Chart paths configuration" width="450"/>
+## Configuration
 
 
-4. Put charts into selected paths
+### Local Chart Files
 
-5. Add optional online chart providers
+If you are using chart files stored on the Signal K Server you will need to add the locations where the chart files are stored so the plugin can generate the chart metadata.
```

Comment:
I think this section should read like
- the plugin needs to find your local chart files
- you can either put the files in the default location xxx or add configuration entries for additional directories

Now it sounds like you need to configure the locations, that is not actually required.